### ISSUE TITLE
Element visibility is not triggered by hovering another element

### ISSUE THREAD

**Issue by HPetra** [php-webdriver/php-webdriver#477]
### What are you trying to achieve? (Expected behavior)
Trigger an appearance of an element, by hovering an another element on it.

### What do you get instead? (Actual behavior)
PHP Fatal error:  Uncaught exception 'Facebook\WebDriver\Exception\NoSuchElementException' with message 'no such element: Unable to locate element: {"method":"xpath","selector":"//div[contains(@style,'background-color: darkgreen')]"}

### How could the issue be reproduced? (Steps to reproduce)
```php
$capabilities = DesiredCapabilities::htmlUnitWithJS();
    {
        // For Chrome
        $options = new ChromeOptions();
        $prefs = array('download.default_directory' => '/tmp');
        $options->setExperimentalOption('prefs', $prefs);
        $capabilities = DesiredCapabilities::chrome();
        $capabilities->setCapability(ChromeOptions::CAPABILITY, $options);
    }
$driver = RemoteWebDriver::create($host, $capabilities, 5000);

$driver->get('http://react-dnd.github.io/react-dnd/examples-dustbin-multiple-targets.html');
$driver->navigate()->to('http://react-dnd.github.io/react-dnd/examples-dustbin-multiple-targets.html');

$dragBox = $driver->findElement(WebDriverby::xpath("//div[@data-reactid='105']"));

$dropBox = $driver->findElement(WebDriverBy::xpath("//div[@data-reactid='96']"));
$driver->action()
    ->clickAndHold($dragBox)
    ->moveToElement($dropBox)
    ->perform();

$driver->wait(15, 1000)->until(
    WebDriverExpectedCondition::presenceOfElementLocated(WebDriverBy::xpath("//div[contains(@style,'background-color: darkgreen')]"))
);

$driver->action()
    ->release($dropBox)
    ->perform();
```

### Details

* PHPUnit: 5.0.9
* ChromeDriver: 2.32.498537
* PHP version: 5.6.29
* Operating system: MacOS
* Browser used + version: Chrome,  60.0.3112.113

**Comment by matthiaskrieft** [comment]
Same problem here. Is there any idea we could try?

**Comment by OndraM** [comment]
Hi Matthias, could you also provide steps to reporduce the problem,
versions etc.? This would help debugging the issue. Thanks a lot!

Dne 16. 10. 2017 8:44 dop. napsal uživatel "Matthias Krieft" <
notifications@github.com>:

**Comment by matthiaskrieft** [comment]
@OndraM I was able to reproduce the steps from @HPetra

**Comment by OndraM** [comment]
Using exactly the same code? 

If so, could you please try with the latest stable chrome (61) and chromedriver (2.33)?

Also providing logs from selenium server might help.

Thanks :)

**Comment by matthiaskrieft** [comment]
I just retried with the requested versions. Here comes the output. example.php contains just the code from above:

`$ phpunit example.php 
PHP Fatal error:  Uncaught exception 'Facebook\WebDriver\Exception\NoSuchElementException' with message 'no such element: Unable to locate element: {"method":"xpath","selector":"//div[contains(@style,'background-color: darkgreen')]"}
  (Session info: chrome=61.0.3163.100)
  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)' in /www/selenium/vendor/facebook/webdriver/lib/Exception/WebDriverException.php:102
Stack trace:
#0 /www/selenium/vendor/facebook/webdriver/lib/Remote/HttpCommandExecutor.php(320): Facebook\WebDriver\Exception\WebDriverException::throwException(7, 'no such element...', Array)
#1 /www/selenium/vendor/facebook/webdriver/lib/Remote/RemoteWebDriver.php(535): Facebook\WebDriver\Remote\HttpCommandExecutor->execute(Object(Facebook\WebDriver\Remote\WebDriverCommand))
#2 /www/selenium/vendor/facebook/webdriver/lib/Remote/RemoteWebDriver.php(176): Facebook\WebDriver\Remote\RemoteWebDriver->execute('findElement', Array)
#3 /www/selenium/vendor/facebook/ in /www/selenium/vendor/facebook/webdriver/lib/Exception/WebDriverException.php on line 102

Fatal error: Uncaught exception 'Facebook\WebDriver\Exception\NoSuchElementException' with message 'no such element: Unable to locate element: {"method":"xpath","selector":"//div[contains(@style,'background-color: darkgreen' in /www/selenium/vendor/facebook/webdriver/lib/Exception/WebDriverException.php on line 102

Facebook\WebDriver\Exception\NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//div[contains(@style,'background-color: darkgreen')]"}
  (Session info: chrome=61.0.3163.100)
  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64) in /www/selenium/vendor/facebook/webdriver/lib/Exception/WebDriverException.php on line 102

Call Stack:
    0.0041     384656   1. {main}() /usr/local/bin/phpunit:0
    0.1204   11577168   2. PHPUnit_TextUI_Command::main() /usr/local/bin/phpunit:513
    0.1204   11577808   3. PHPUnit_TextUI_Command->run() phar:///usr/local/bin/phpunit/phpunit/TextUI/Command.php:105
    0.1246   11590944   4. PHPUnit_Runner_BaseTestRunner->getTest() phar:///usr/local/bin/phpunit/phpunit/TextUI/Command.php:128
    0.1246   11590992   5. PHPUnit_Runner_BaseTestRunner->loadSuiteClass() phar:///usr/local/bin/phpunit/phpunit/Runner/BaseTestRunner.php:67
    0.1246   11591520   6. PHPUnit_Runner_StandardTestSuiteLoader->load() phar:///usr/local/bin/phpunit/phpunit/Runner/BaseTestRunner.php:123
    0.1251   11742680   7. PHPUnit_Util_Fileloader::checkAndLoad() phar:///usr/local/bin/phpunit/phpunit/Runner/StandardTestSuiteLoader.php:39
    0.1251   11742832   8. PHPUnit_Util_Fileloader::load() phar:///usr/local/bin/phpunit/phpunit/Util/Fileloader.php:38
    0.1255   11760512   9. include_once('/www/selenium/example.php') phar:///usr/local/bin/phpunit/phpunit/Util/Fileloader.php:56
   16.9860   12859408  10. Facebook\WebDriver\WebDriverWait->until() /www/selenium/example.php:35`

This is the output of the selenium server:
`16:12:57.585 INFO - Binding default provider to: org.openqa.selenium.chrome.ChromeDriverService
16:12:57.587 INFO - Found handler: org.openqa.selenium.remote.server.BeginSession@689cc686
16:12:57.589 INFO - /session: Executing POST on /session (handler: BeginSession)
16:12:57.594 INFO - Capabilities are: Capabilities {browserName=chrome, chromeOptions={binary=, prefs={download.default_directory=/tmp}, }, platformName=ANY, platform=ANY, }
16:12:57.595 INFO - Capabilities {browserName=chrome, chromeOptions={binary=, prefs={download.default_directory=/tmp}, }, platformName=ANY, platform=ANY, } matched class org.openqa.selenium.remote.server.ServicedSession$Factory (provider: org.openqa.selenium.chrome.ChromeDriverService)
16:12:57.595 INFO - Capabilities {browserName=chrome, chromeOptions={binary=, prefs={download.default_directory=/tmp}, }, platformName=ANY, platform=ANY, } matched class org.openqa.selenium.remote.server.ServicedSession$Factory (provider: org.openqa.selenium.chrome.ChromeDriverService)
16:12:57.596 INFO - Capabilities {browserName=chrome, chromeOptions={binary=, prefs={download.default_directory=/tmp}, }, platformName=ANY, platform=ANY, } matched class org.openqa.selenium.remote.server.ServicedSession$Factory (provider: org.openqa.selenium.chrome.ChromeDriverService)
Starting ChromeDriver 2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2) on port 31641
Only local connections are allowed.
16:12:59.179 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:12:59.180 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/url (handler: ServicedSession)
16:12:59.182 INFO - To upstream: {"url":"http:\/\/react-dnd.github.io\/react-dnd\/examples-dustbin-multiple-targets.html"}
16:13:13.254 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":0,"value":null}
16:13:13.272 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:13.273 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/url (handler: ServicedSession)
16:13:13.285 INFO - To upstream: {"url":"http:\/\/react-dnd.github.io\/react-dnd\/examples-dustbin-multiple-targets.html"}
16:13:13.931 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":0,"value":null}
16:13:13.934 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:13.935 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:13.938 INFO - To upstream: {"using":"xpath","value":"\/\/div[@data-reactid='105']"}
16:13:13.974 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":0,"value":{"ELEMENT":"0.601311298402988-1"}}
16:13:13.994 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:13.995 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:13.997 INFO - To upstream: {"using":"xpath","value":"\/\/div[@data-reactid='96']"}
16:13:14.040 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":0,"value":{"ELEMENT":"0.601311298402988-2"}}
16:13:14.047 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:14.048 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/moveto (handler: ServicedSession)
16:13:14.050 INFO - To upstream: {"element":"0.601311298402988-1"}
16:13:14.107 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":0,"value":null}
16:13:14.109 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:14.109 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/buttondown (handler: ServicedSession)
16:13:14.112 INFO - To upstream: 
16:13:14.115 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":0,"value":null}
16:13:14.117 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:14.117 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/moveto (handler: ServicedSession)
16:13:14.119 INFO - To upstream: {"element":"0.601311298402988-2"}
16:13:14.354 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":0,"value":null}
16:13:14.359 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:14.359 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:14.361 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:14.463 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:15.580 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:15.584 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:15.593 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:15.631 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:16.640 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:16.641 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:16.643 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:16.664 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:17.667 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:17.667 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:17.669 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:17.699 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:18.714 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:18.715 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:18.717 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:18.742 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:19.745 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:19.745 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:19.747 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:19.768 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:20.777 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:20.778 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:20.781 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:20.808 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:21.815 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:21.815 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:21.817 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:21.835 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:22.840 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:22.841 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:22.843 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:22.856 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:23.863 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:23.864 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:23.866 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:23.878 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:24.883 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:24.883 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:24.885 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:24.894 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:25.902 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:25.902 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:25.906 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:25.933 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:26.954 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:26.958 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:26.966 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:26.996 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:28.002 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:28.002 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:28.008 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:28.039 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
16:13:29.061 INFO - Found handler: org.openqa.selenium.remote.server.ServicedSession@1327502d
16:13:29.066 INFO - Handler thread for session 547f49ae71810c09ff24bcd59fdbde3a (chrome): Executing POST on /session/547f49ae71810c09ff24bcd59fdbde3a/element (handler: ServicedSession)
16:13:29.076 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@style,'background-color: darkgreen')]"}
16:13:29.117 INFO - To downstream: {"sessionId":"547f49ae71810c09ff24bcd59fdbde3a","status":7,"value":{"message":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(@style,'background-color: darkgreen')]\"}\n  (Session info: chrome=61.0.3163.100)\n  (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.12.6 x86_64)"}}
`

**Comment by matthiaskrieft** [comment]
@OndraM does that help?

**Comment by schroeder** [comment]
+1

**Comment by OndraM** [comment]
Hi, I wasn't able to make dragAndDrop working in chrome as well. However I'm not able to tell if its an issue of the Chromedriver, Selenium itself or php-webdriver. I will have to dig it this deeper, but sorry, no ETA here right now, #469 will now be a priority. Hope you understand.

**Comment by OndraM** [comment]
You may also try reporting the issue to https://github.com/SeleniumHQ/selenium/issues or try searching if it wasn't already reported. However reproducible usecase in different language (eg. java) would be required so that folks ad Selenium project can have a look.

**Comment by OndraM** [comment]
Closing, as it may be resolved by #469. Feel free to open new issue with reproducible test case if the problem still persist. Thanks :)
